src/app/core/submission/workflowitem-data.service.ts

```diff
@@ -35,4 +39,38 @@ export class WorkflowItemDataService extends DataService<WorkflowItem> {
     super();
   }
 
+  /**
+   * Delete an existing Workspace Item on the server
+   * @param id The Workspace Item's id to be removed
+   * @return an observable that emits true when the deletion was successful, false when it failed
+   */
+  delete(id: string): Observable<boolean> {
+    return this.deleteWFI(id, true)
+  }
+
+  sendBack(id: string): Observable<boolean> {
+    return this.deleteWFI(id, false)
+  }
+
+  private deleteWFI(id: string, expunge: boolean): Observable<boolean> {
```

Comment:
Tiny thing, could we add TypeDocs to this method as it's called by both functions above.  I think it's worth describing here that the WFI is deleted either way, but in one situation the result is the WorkSpaceItem still exists, and in the other it's expunged.

Reply:
Added missing TypeDoc